src/index.html

```diff
@@ -10,7 +10,47 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="get" 
+      data-qa="big"
+      class="form"
+    >
+      <img 
+        src="./images/Search.svg" 
+        alt="search" 
+        class="search_svg"
+      >
+      
+      <input
+        class="search_bar search__large"
+        type="text" 
+        name="search_bar" 
```

Comment:
```suggestion
        name="searchBar" 
```
![image](https://user-images.githubusercontent.com/96409563/223065977-8e7136a4-3b86-45cd-9e2e-63f2c51bc865.png)